### PR TITLE
German translation contributor chapter 01

### PR DESCRIPTION
**Pull request by RJPlog**
Hi, I started translation again, here the translation of the first chapter of contributor

### CODE REVIEW

**Comment by rrrutledge** [comment]
Oh sorry just saw this!  Will bring up in the community.

File: contributor/de/01-introduction-article-de.asciidoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Ein anschauliches Beispiel dafür ist die im Einführungsvideo gezeigte Gast/Gastgeber Analogie.
+
+Das vierte Kapitel beschreibt die praktischen Dinge, die getan werden müssen um deinen Beitrag zum Erfolg zu führen - die _Mechanik des Mitwirkens_.
+Wir geben praktische Tipps, die dir bei der Vorbereitung auf die Arbeit an einem Beitrag, während der Entwicklung und auch bei der Pull-Anfrage nützen.
+
+Nachdem wir die persönlichen Aspekte, die der Zusammenarbreit und die technischen Aspekte der Contributor Rolle betrachtet haben, gehen wir im fünften Kaptiel darauf ein welcher Nutzen dem Aufwand beim Mitwirken gegenüber steht. Wir zeigen den Nutzen aus verschiedenen Sichten, deiner, die deines Teams und aus der Sicht des gesamten Unternehmens.
+
+Das letzte Kapitel wird nochmal zusammenfassen, was es bedeutet, ein InnerSource Contributor zu sein.
+Wir zeigen auf, wie du deine Kenntnisse über InnerSource sowohl mit unseren online Videos und Artikeln, als auch durch direkte Einbindung in die InnerSource Community selbst, vertiefen kannst
```
Comment (lenucksi) [inline contributor/de/01-introduction-article-de.asciidoc:63]:
```suggestion
Wir zeigen auf, wie du deine Kenntnisse über InnerSource sowohl mit unseren online Videos und Artikeln, als auch durch direkte Einbindung in die InnerSource Community selbst, vertiefen kannst.
```

**Review by lenucksi — APPROVED** [review]
That's an excellent translation @RJPlog. I've added two small typos that I found that you can probably just click-in.

Also: Please excuse the late reaction.

**Comment by RJPlog** [comment]
Thx for your reviews! I applied all changes! See you with my next chapter. :-)

**Comment by rrrutledge** [comment]
Very much looking forward to this!

**Comment by lenucksi** [comment]
All of these look great, so it's time for a merge. Thank you again for translating @RJPlog! :D

**Comment by tsadler1988** [comment]
Confirmed this is now on the website.

**Pull request merged**